.idea/check_done.iml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
-    <content url="file://$MODULE_DIR$" />
-    <orderEntry type="jdk" jdkName="Poetry (check_done)" jdkType="Python SDK" />
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/.venv" />
+    </content>
+    <orderEntry type="jdk" jdkName="Python 3.12 (check_done)" jdkType="Python SDK" />
```

Comment:
Contributors should either adhere to the project's IDE configuration by renaming the virtual environment used for the project to match the existing IDE project settings, or not commit this file. This should be reverted.